Dockerfile

```diff
@@ -1,26 +1,27 @@
 # Build the manager binary
-# Build the manager binary
-FROM golang:1.16 as builder
+FROM --platform=${BUILDPLATFORM} golang:1.16 as builder
+
+ARG TARGETARCH
+ARG TARGETOS
 
 WORKDIR /workspace
 # Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
+COPY go.mod go.sum ./
```

Comment:
This would revert the docker caching of the go.mod copy. Please revert this.

Reply:
I don’t think you get much value out of that single-file layer (and `go.mod` and `go.sum` usually change together). But OK, reverted.